lesson2/script.js

```diff
@@ -0,0 +1,27 @@
+    const li = document.createElement('li');
+
+    const a = document.createElement('a');
+    const href = document.createAttribute('href');
+    href.value = "1.html";
+    a.setAttributeNode(href);
+
+    const img = document.createElement('img');
+    const src = document.createAttribute('src');
+    src.value = "bookmark.png";
+    img.setAttributeNode(src);
+    const alt = document.createAttribute('alt');
+    alt.value = "ブックマーク";
```

Comment:
this is good. but, Do you know to have node(img or a that created by createElement method) itself accessible to attribute.

```js
img.alt = "ブックマーク"
```

Reply:
I understand that the node created by the createElement method has attributes, so I could write it simpler.
35bdbfe